### ISSUE TITLE
[5.x] Calling commands from addons

### ISSUE THREAD

**Issue by robdekort** [statamic/cms#9861]
### Bug description

I have an [addon](https://github.com/studio1902/statamic-peak-commands) that adds certain commands to Peak sites. In some of those commands I call other Artisan commands:

1. `Artisan::call('cache:clear');`
2. `Artisan::call('statamic:glide:clear')`

In one command I [call them both](https://github.com/studio1902/statamic-peak-commands/blob/9720dde194b81e7465b0eb1bc0feaa0b464d9408/src/Commands/ClearSite.php#L37-L38).

In v5 Alpha 1 and 2 triggering those commands leads to an error:
<img width="424" alt="Screenshot 2024-04-11 at 09 32 27" src="https://github.com/statamic/cms/assets/69107412/7b6342ef-682c-4535-9724-17e9eb1d7813">

When I replicate those commands in my actual project (and not in an addon) they work without issue. I can replace the `cache:clear` command with `\Illuminate\Support\Facades\Cache::clear()` and that works, however there's no alternative like that for the `statamic:glide:clear` command. 

### How to reproduce

1. Install the v5 alpha.
2. Create an addon.
3. Add a simple command to the addon.
4. Call an artisan command from the addon.

### Environment

```yaml
Environment
Application Name: Statamic Peak Development
Laravel Version: 11.3.1
PHP Version: 8.3.4
Composer Version: 2.7.1
Environment: local
Debug Mode: ENABLED
URL: statamic-peak.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: sqlite
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 4
Sites: 1
Stache Watcher: Enabled
Static Caching: Disabled
Version: 5.0.0-alpha.2 PRO

Statamic Addons
studio1902/statamic-peak-browser-appearance: dev-feature/v5-support
studio1902/statamic-peak-commands: dev-feature/v5-support
studio1902/statamic-peak-seo: dev-feature/v5-support
studio1902/statamic-peak-tools: dev-feature/v5-support
```

### Installation

Starter Kit using via CLI

### Antlers Parser

None

**Comment by duncanmcclean** [comment]
I'm assuming you were doing it inside `php please` command, correct?

Can you try running that same command but with Artisan, so `php artisan statamic:*`?

**Comment by robdekort** [comment]
Correct, and that makes it work Duncan!

**Comment by duncanmcclean** [comment]
Cool, thanks for testing! We probably need to do some more tweaking on our side regarding how `please` works (related: #9729).

**Comment by robdekort** [comment]
Cool, glad you're on the right path!

**Comment by robdekort** [comment]
![Screenshot 2024-04-18 at 09 51 44](https://github.com/statamic/cms/assets/69107412/76f1c1c7-f9e3-4832-a441-e1b00f55d64e)

On Alpha 4 is still notice an issue with the glide command. Notice in the screenshot above that:
* `php artisan statamic:peak:clear-site` works
* `php please peak:clear-site` does not work

**Comment by duncanmcclean** [comment]
Not saying it's the fix but can you try `Artisan::call('peak:clear-site')` (so without the `statamic:prefix`)?

**Comment by robdekort** [comment]
Do you mean `Artisan::call('glide:clear')`?

**Comment by duncanmcclean** [comment]
Yeah

**Comment by robdekort** [comment]
Yep, that works!

**Comment by robdekort** [comment]
Should I just use that and consider the case closed?

**Comment by duncanmcclean** [comment]
Nah, leave it open, it should work with the `statamic:` prefix.

**Comment by jasonvarga** [comment]
Nah I'd expect `Artisan::call('statamic:glide:clear')` to be what you need to run and **not** `Artisan::call('glide:clear')`. We'll sort it out. Thanks!

**Comment by robdekort** [comment]
Hokey dokey. Thanks guys.

**Comment by jasonvarga** [comment]
Fixed in #9926
